core/player/src/expressions/parser.ts

```diff
@@ -899,37 +911,51 @@ export default function parseExpression(expr: string): ExpressionNode {
 
   const nodes = [];
 
-  while (index < length) {
-    const chIndex = exprICode(index);
+  try {
+    while (index < length) {
+      const chIndex = exprICode(index);
+
+      // Expressions can be separated by semicolons, commas, or just inferred without any
+      // separators
+      if (chIndex === SEMCOL_CODE || chIndex === COMMA_CODE) {
```

Comment:
Would there be any worth to having the kind of characters that are ignored be configurable?

Reply:
That may make it pretty difficult to define a proper bnf grammar for expressions. But that might be worth considering as a future change